echopype/utils/prov.py

```diff
@@ -17,9 +18,10 @@ def echopype_prov_attrs(process_type: ProcessType) -> Dict[str, str]:
     process_type : ProcessType
         Echopype process function type
     """
+    # TODO: change hard coded 0.6.0 after release
     prov_dict = {
         f"{process_type}_software_name": "echopype",
-        f"{process_type}_software_version": ECHOPYPE_VERSION,
+        f"{process_type}_software_version": "0.6.0",  # ECHOPYPE_VERSION,
```

Comment:
We need to change this after the release. It is hard coded to 0.6.0 right now so that tests pass.

Reply:
@emiliom if you don't mind, I would like to leave this as unresolved so that we can refer to it in the future. Is that ok?